packages/@aws-cdk/aws-lambda-python/lib/types.ts

```diff
@@ -86,4 +86,17 @@ export interface BundlingOptions {
    * @default - Based on `assetHashType`
    */
   readonly assetHash?: string;
+
+  /**
+   * Where to mount the specified volumes from
+   * @see https://docs.docker.com/engine/reference/commandline/run/#mount-volumes-from-container---volumes-from
+   * @default - no containers are specified to mount volumes from
+   */
+  readonly volumesFrom?: string[];
+
+  /**
+   * Additional commands executed in the bundling container after running dependency installation
+   * @default - no additional custom commands are executed
+   */
+  readonly customCommands?: string[]
```

Comment:
Well this poped up out of nowhere. 😅 

FYI: Lambda functions have converged on using [CommandHooks](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_lambda_nodejs.BundlingOptions.html#commandhooks)

Reply:
this was required to make the test work